### DISCUSSION TITLE
Add a per person limit to shop(s)

### DISCUSSION THREAD

**Discussion by zerafox** [gameserverapp/Platform#652] [Suggestions]
Add an option to both Shop Packs and Supporter Tiers that allow a limit of purchases for individual person(s).

Right now there is only settings available on a global scale, either a single or multiple people can obtain a pack/tier X amount of times before it's hidden from everybody.

It'd be nice to say "Each person gets X amount then it hides from said person".

This would be ideal for say single or 2-3 time use Starting Packs for example.

**Comment by mvaessen** [comment]
hi, all limits for shop packs are personal limits. There is only 1 global limit, which is a feature :)
There is currently no plan to make supporter tiers limited too.
